src/components/ModalWithConfirmation.js

```diff
@@ -0,0 +1,19 @@
+import Modal from "./Modal.js";
+
+export default class ModalWithConfirmation extends Modal {
+  constructor(modalSelector) {
+    super(modalSelector);
+    this._confirmationModalForm = this._modal.querySelector(".modal__form");
+  }
+  setSubmitAction(Action) {

```

Comment:
`Action`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)